anton-melyakov/homework1/CV anton/cv frontpage.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Anton Melyakov`s CV - absolutely master(bash maystor)</title>
+	<link rel="stylesheet" type="text/css" href="main.css">
+</head>
+<body>
+<!-- This is the main heading of the site -->
+	<header id="site-header">
+		<img id="cv-logo" src="cv_logo.jpg" alt="picture of me">
+		<h1 id="cv-title">Anton Melyakov<br> CV and Bio</h1>
+		<nav id="top-navigation">
+			<a href="https://www.facebook.com/profile.php?id=100010838534570&fref=ts">Facebook page</a>
+			<a href="">Professional experience</a>
+			<a href="">Photo and video</a>
+			<a href="htthttp://downpourbg.com/">The site project</a>
```

Comment:
@AntonMelyakov 
The second and third link, are empty. You also want to change "htthttp://downpourbg.com/" to "http://downpourbg.com/" ^_^